netcdf_to_gltf_converter/gltf/builder.py

```diff
@@ -14,11 +14,11 @@
 from netcdf_to_gltf_converter.utils.arrays import float32_array
 
 PADDING_BYTE = b"\x00"
-ROTATION_MATRIX = [1, 0, 0, 0, 
-                   0, 0, -1, 0, 
-                   0, 1, 0, 0, 
-                   0, 0, 0, 1]
-"""Rotation matrix to flip the y and z axes."""
+ROTATION_MATRIX = [-0.9998477, 0, 0.0174524, 0,
```

Comment:
I have an explanation why it is supposed to change written down here: 
[Flip y and z axes such that y is up (instead of z) 179 degrees.pdf](https://github.com/Deltares-research/netcdf_to_gltf_converter/files/15388914/Flip.y.and.z.axes.such.that.y.is.up.instead.of.z.179.degrees.pdf)

Reply:
Thanks Ralph :D